### PR TITLE
Console errors fix for Data reference

### PR DESCRIPTION
**Pull request by sivaramakrishnatumma**
Fix for two console errors related to "Each child in a list should have a unique 'key' prop"

### CODE REVIEW

**Review by JEAxero — COMMENTED** [review]
Since there were no changes to package.json, do you know what changed in package-lock.json and why it got updated? Just curious since I don't think anything inthe other changes you made would have caused it to change.

**Comment by sivaramakrishnatumma** [comment]
> Since there were no changes to package.json, do you know what changed in package-lock.json and why it got updated? Just curious since I don't think anything in the other changes you made would have caused it to change.

I have pulled the latest main branch code and did npm install which caused changes in package-lock.json file

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,12110 @@
 {
   "name": "react-sdk",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (JEAxero) [inline package-lock.json:4]: Any idea why lockFileVersion got bumped to 2?

**Comment by sivaramakrishnatumma** [comment]
I guess this happened because I was having node version of v16.13.2 and npm version of 8.1.2. Let me undo the changes for the package-lock.json file.

**Comment by JEAxero** [comment]
> I guess this happened because I was having node version of v16.13.2 and npm version of 8.1.2. Let me undo the changes for the package-lock.json file.

Yeah. Constellation 8.6 and 8.7 don't fully work with node 16 yet. Preferred settings are something like:
node v14.18.*
npm 6.14.*

Thanks!

**Review by JEAxero — APPROVED** [review]
Thanks!

**Pull request merged**